### ISSUE TITLE
Home Assistant Logs not working when this component is enabled

### ISSUE THREAD

**Issue by holdestmade** [robbrad/UKBinCollectionData#464]
### Home Assistant Version

2023.11.x

### Installation Method

Using HACS

### Name of Council (if relevant)

East Riding of Yorkshire

### Issue Information

Bit of a strange one, my HA logs stop working after startup. I disabled all my integrations and enabled them one by one and found this one to be the cause. When this component is enabled, my normal HA logs do not appear. 

I have also tested this using the log write service and while this is enabled, it does not show up in the logs

Any reasons you can see that this happens ?

Thanks

### Verification

- [X] I searched for similar issues at https://github.com/robbrad/UKBinCollectionData/issues?q=is:issue and found no duplicates
- [X] If trying to add a specific council, I've checked it is listed as working at https://robbrad.github.io/UKBinCollectionData/3.11/
- [X] I have provided a detailed explanation of the issue as well as steps to replicate the issue
- [X] I understand that this project is run by volunteer contributors therefore completion of this issue cannot be guaranteed

**Comment by robbrad** [comment]
@OliverCullimore : didnt you experiance this when we tested it?

**Comment by mrw298** [comment]
I think I've seen the same/a similar issue, after the call here to ukbcd.run the logging stops (don't see the line underneath)

```
  async def _async_update_data(self):
        async with async_timeout.timeout(30) as cm:
            _LOGGER.info(f"{LOG_PREFIX} UKBinCollectionApp Updating")

            data = await self.hass.async_add_executor_job(self.ukbcd.run)

            _LOGGER.info(f"{LOG_PREFIX} UKBinCollectionApp: {data}")
```

**Comment by OliverCullimore** [comment]
@robbrad can't recall noticing that no, may have been that the UKBCD python errors weren't being output not the HA ones but was a while ago now

**Comment by dandantheitman** [comment]
I think you're on the right lines with the logger config causing the problems. I don't know python so don't know for sure what's going on or why the custom config options are needed, but without them everything seems to work fine for me.

I updated the following:
https://github.com/robbrad/UKBinCollectionData/blob/55ab3781b2db890c9f862e7a3f48bfa497b93031/uk_bin_collection/uk_bin_collection/get_bin_data.py#L34-L34
to just be:
```
LOGGER = logging.getLogger(None)
```
This stops the custom logger config ever being used. Once I made that change locally and restarted home assistant my logging works perfectly again.
